components/Meta.jsx

```diff
@@ -0,0 +1,49 @@
+import Head from 'next/head';
+import PropTypes from 'prop-types';
+
+const SITE_TITLE = 'Build | Olas';
+const SITE_DESCRIPTION = 'Explore paths to build on Olas. Simplify your path to contributing and earning OLAS rewards.';
+const SITE_URL = 'https://build.olas.network/';
+const SITE_DEFAULT_IMAGE_URL = `${SITE_URL}/images/metatags-image.png`;
+
+const Meta = ({ title, description, url }) => {
+  const pageTitle = title ? `${title} | ${SITE_TITLE}` : SITE_TITLE;
+  return (
+    <Head>
+      <title>{pageTitle}</title>
+
+      <meta name="title" content={pageTitle} />
+      <meta name="description" content={description || SITE_DESCRIPTION} />
+
+      <meta property="og:type" content="website" />
+      <meta property="og:url" content={url || SITE_URL} />
```

Comment:
maybe would be easier to do it like you do with title, so we don't repeat the origin url every time?: 
```
content={`${SITE_URL}/${path}}
```
(renamed url to path so it's clearer it's not the base url)